src/features/layers_panel/index.tsx

```diff
@@ -1,5 +1,14 @@
-import { MapLayerPanel } from './components';
+import { lazily } from 'react-lazily';
+import { i18n } from '~core/localization';
+import { MIN_HEIGHT } from './constants';
+import type { PanelFeatureInterface } from 'types/featuresTypes';
 
-export function MapLayersList() {
-  return <MapLayerPanel />;
-}
+const { PanelContent } = lazily(() => import('./components'));
```

Comment:
it feels like lazy loading (lazily + <Suspense fallback={null}> wrapping) should be done on place of usage (Map.tsx)

Reply:
if we export an object from feature (with type of feature interface), it can't be lazy loaded using `lazily` out of the box, we'll need to write a wrapper
but exporting it unconditionally isn't the options
so that's the middle ground

Reply:
ah, I see 🤔